### DISCUSSION TITLE
Guys, I've thought about it, and the only way forward is material you!

### DISCUSSION THREAD

**Discussion by bmaroti9** [bmaroti9/Overmorrow#22] [Announcements]
<img src="https://github.com/user-attachments/assets/6177fba6-749e-44ad-9d82-a611b1a61445" width="200">
<img src="https://github.com/user-attachments/assets/d4cda5cb-b681-451c-8016-2b3b9039b6d4" width="200">

I have started to redesign the app with material design elements. 
I still want to keep the colorful colormode but i have no idea if there even are material apps with a colorfull background instead of light and dark, so i haven't done anything about it yet.
Also there are now there are going to be network images from unsplash.

**Comment by Kreuger** [comment]
Might I suggest having Material You as just another theme option? Ive seen that in some apps.

**Comment by bmaroti9** [comment]
Yeah, i know there should be something like this. But i'm still not sure how i should do it yet.
I basically have to options:
1. just have the image and display the ui on top but with the material you accent colors (ignoring the image colors). this is a much easier solution, but it could look bad to have say a red sunset image, with green accent colors
2. somehow try to get images that are the same color as the material-you theme. i explain my idea better in #9. I have experimented with this a little bit, and it works fine with blue, yellow (basically just clouds and sky) okay with green (some plant or scenery) but abysmally with red or purple (basically nothing exists).

If i can, i definitely want to do the 2nd one, but i'm still questioning the quality, and the relevancy of the unsplash images, if i try to get specific colors.

**Comment by Kreuger** [comment]
Im not sure how to help you with that one. Personally I dont see a point in using MY. I got this app because I love how beautiful it looks as is.

**Comment by bmaroti9** [comment]
Ooooh, I totally misunderstood what you said. Right, material-you as another theme option. Are you talking about the colors, or the widgets, or both?

**Reply by Kreuger** [reply]
Both.
